pandas/tests/arrays/categorical/test_analytics.py

```diff
@@ -35,6 +35,16 @@ def test_min_max(self):
         assert _min == "d"
         assert _max == "a"
 
+    def test_min_max_empty(self):
+        # GH 30227
+        cat = Categorical([], categories=list("ABC"), ordered=True)
+
+        result = cat.min()
+        assert isinstance(result, float) and np.isnan(result)
```

Comment:
you will have to compare vs NaN and pd.NaT here (you can use ``is``), add this to the parametization